pkg/security/probe/stateful_probe_excluder.go

```diff
@@ -67,6 +67,10 @@ func (af *availableFunctionsBasedExcluder) ShouldExcludeFunction(name string, pr
 		return false
 	}
 
+	if strings.HasPrefix(prog.SectionName, "uprobe") || strings.HasPrefix(prog.SectionName, "uretprobe") {
```

Comment:
```suggestion
	if strings.HasPrefix(prog.SectionName, "uprobe/") || strings.HasPrefix(prog.SectionName, "uretprobe/") {
```
maybe include the / just in case ? Otherwise LGTM